latex/cardano-ledger.tex

```diff
@@ -36,20 +36,65 @@
 \includeAgda{Ledger/Chain}
 
 \section{Properties}
-\includeAgda{Ledger/Utxo/Properties}
+This section presents the properties of the ledger that we have formally proved in
+Agda or plan to do so in the near future.  We indicate in which Agda module each
+property is formally stated and (possibly) proved. A ``Claim'' is a property that is
+not yet proved, while a ``Theorem'' is one for which we have a formal proof.
+
+\subsection{Preservation of Value}
+\inputAgda{Ledger/Ledger/Properties/PoV}
+\inputAgda{Ledger/Utxo/Properties/PoV}
+\inputAgda{Ledger/Certs/Properties/PoV}
+\inputAgda{Ledger/Certs/Properties/PoVLemmas}
+
+\subsection{Invariance Properties}
```

Comment:
Maybe this subsection is missing a `govDepsMatch`-independent introduction to what is meant by invariance properties (given a predicate P on states and a rule like Ledger then blablabla), and this introduction corresponds to a subsubsection.

Reply:
I agree, although I think an extra subsubsection for a single introductory sentence is overkill.